automated-comments: add initial disapprove comments for java/two-fer (#1120)
exercism/website-copy: automated-comments/java/general/avoid_hard_coded_test_cases.md

This solution has hard-coded some of the test data.

> Generally, solutions should not be relying on the specific data being tested.

Reviewer: ```suggestion
Generally, solutions should not rely on the specific data being tested.
```

> Instead, the solutions should solve the problem generally for any input rather

Reviewer: ```suggestion
Instead, the solutions should solve the problem generally, for any input.
```

> than simply accounting for the exact data in the tests.

Reviewer: ```suggestion
```

> Please try again without hard-coding the test data into your solution.

Reviewer: ```suggestion
Remove the hard-coded test data.
```
